fvtest/compilertriltest/JitTest.hpp

```diff
@@ -40,7 +42,8 @@ class JitTest : public ::testing::Test
 
    static void SetUpTestCase()
       {
-      ASSERT_TRUE(initializeJit()) << "Failed to initialize the JIT.";
+      auto initSuccess = initializeJitWithOptions((char*)"-Xjit:acceptHugeMethods,enableBasicBlockHoisting,omitFramePointer,useIlValidator,paranoidoptcheck");
+      ASSERT_TRUE(initSuccess) << "Failed to initialize the JIT.";
```

Comment:
Yes, it's just there to match JitBuilder's init code.

Until we have a backend that provides more control to test writers, I think it's best to keep things consistent.